app/src/main/java/org/gdg/frisbee/android/activity/ArrowActivity.java

```diff
@@ -66,6 +66,7 @@
 
 public class ArrowActivity extends GdgNavDrawerActivity {
 
+    private static final Charset CHARSET = Charset.forName("US-ASCII");
```

Comment:
Using a more meaningful name like CHARSET_US_ASCII would be more clear. CHARSET is pretty generic for a final like this.